src/pages/Lend/LoanSearchPage.vue

```diff
@@ -70,7 +70,7 @@ export default {
 				});
 		}
 	},
-	created() {
+	mounted() {
```

Comment:
Why was this change needed?

Reply:
Analytics calls can only execute in the client, so it's a more explicit location to attempt the call. More preference than necessity.